src/mono/mono/utils/mono-context.c

```diff
@@ -534,10 +534,28 @@ mono_sigctx_to_monoctx (void *sigctx, MonoContext *mctx)
 #endif
 #ifdef __linux__
 	struct fpsimd_context *fpctx = (struct fpsimd_context*)&((ucontext_t*)sigctx)->uc_mcontext.__reserved;
-	int i;
 
-	g_assert (fpctx->head.magic == FPSIMD_MAGIC);
-	for (i = 0; i < 32; ++i)
+	size_t size = 0;
+    do
```

Comment:
This should use mono coding conventions.

Reply:
Fixed.